### ISSUE TITLE
PP-DocLayout_plus-L: Why is Train.num_classes set to 11 while official_categories defines 20 classes?

### ISSUE THREAD

**Issue by kennyHyunSeokCho** [PaddlePaddle/PaddleX#4467]
Hi PaddleX team,

While preparing to fine-tune PP-DocLayout_plus-L, I noticed a mismatch between the YAML config and the official category list.
	•	YAML config:
PaddleX/paddlex/configs/modules/layout_detection/PP-DocLayout-L.yaml
`
Train:
  num_classes: 11
  epochs_iters: 100
  batch_size: 1
  learning_rate: 0.0001
  pretrain_weight_path: https://paddle-model-ecology.bj.bcebos.com/paddlex/official_pretrained_model/PP-DocLayout_plus-L_pretrained.pdparams
  warmup_steps: 100
  resume_path: null
  log_interval: 10
  eval_interval: 1
`

•Official categories for PP-DocLayout_plus-L:
PaddleX/paddlex/repo_apis/PaddleDetection_api/object_det/official_categories.py
This lists 20 classes, e.g.

`
"PP-DocLayout_plus-L": [
    {"name": "paragraph_title", "id": 0},
    {"name": "image", "id": 1},
    {"name": "text", "id": 2},
    ...
    {"name": "reference_content", "id": 19},
]
`

Questions
1.For training/fine-tuning PP-DocLayout_plus-L, should Train.num_classes be 20 to match official_categories.py?
2.Is the 11 in PP-DocLayout-L.yaml intentional (e.g., a config targeting a reduced/11-class dataset), while the pretrained weights and official_categories.py target the full 20-class taxonomy?
3.If 20 is the correct value, are there any additional places we should update besides Train.num_classes (e.g., label schema in the dataset config, heads, or any mapping files)?

What we’re trying to do
We plan to fine-tune PP-DocLayout_plus-L using the provided pretrained weights:
PP-DocLayout_plus-L_pretrained.pdparams

Before starting, we want to confirm the correct num_classes to avoid a mismatch between the detection head and the label set.

Thanks in advance for the clarification!

**Comment by cuicheng01** [comment]
Hello, PP-DocLayout_plus-L does indeed have 20 classes, but the sample dataset we provided contains 11 classes. To ensure proper training, we set the class number to 11 in the configuration. However, rest assured that inference will still be performed on 20 classes. For training, you can change it to the actual number of classes you have.

**Comment by kennyHyunSeokCho** [comment]
Thank you!
